config/locales/es-MX.yml

```diff
@@ -286,6 +286,55 @@ es-MX:
       solidus:
         long: '%B %d, %Y %-l:%M %p'
         short: "%b %-d '%y %-l:%M%P"
+  date:
```

Comment:
these are already provided by rails-i18n (https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/es-MX.yml#L10) and shouldn't be included here. sorry for the late reply 🙇  but could you please remove this `date` block?

Reply:
done